docs/changelog.rst

```diff
@@ -14,12 +14,19 @@ Breaking Changes
 ~~~~~~~~~~~~~~~~
 - ``CreatorCode.status`` and ``CreatorCode.disabled`` have been removed, since they where never returned by this endpoint. Disabled creator codes always raise :class:`fortnite_api.NotFound` when trying to fetch them.
 - ``CreatorCode.verified`` has been removed, since it isn't used within the affiliate system. It always returns ``False``.
+- All enums now use an internal "Enum-like" class to handle unknown values, instead of the built-in ``enum.Enum``. This potentially breaks type checks, but it should not break core functionality.
```

Comment:
```suggestion
- All enums now use an internal "Enum-like" class to handle unknown values, instead of the built-in :class:`py.enum.Enum`. This potentially breaks type checks, but does not break core functionality or change the enum interface; you can use them the same.
```